### ISSUE TITLE
No module named '_ITKCommonBasePython'

### ISSUE THREAD

**Issue by jannessm** [InsightSoftwareConsortium/ITK#2407]
### Description

While trying to use the ITK-Ultrasound package, I noticed that in version 5.2rc03 occurs an import error in ITKCommonBasePython.py

### Steps to Reproduce

1. Setup env with the following packages:
```
itk                             5.2rc3
itk-bsplinegradient             0.2.3
itk-core                        5.2rc3
itk-filtering                   5.2rc3
itk-higherorderaccurategradient 1.0.8
itk-io                          5.2rc3
itk-numerics                    5.2rc3
itk-registration                5.2rc3
itk-segmentation                5.2rc3
itk-splitcomponents             2.0.1
itk-strain                      0.3.1
itk-ultrasound                  0.3
```
2. run 
```python
import itk
import itkConfig
itk.CurvilinearArraySpecialCoordinatesImage[itk.SS, 2]
```

### Expected behavior

no Error

### Actual behavior

```none
lib/python3.6/site-packages/itk/ITKCommonBasePython.py in <module>
     13 
     14 
---> 15 from . import _ITKCommonPython
     16 
     17 

ImportError: attempted relative import with no known parent package
```

### Reproducibility

100%

### Versions

5.2rc03

### Environment

MacOS, Python 3.6

**Comment by thewtex** [comment]
@jannessm thanks for testing, the detailed report, and package versions. I suspect this relates to updates to some of the dependent packages.

**Comment by thewtex** [comment]
This issue was addressed with `itk-ultrasound-0.4.0`. If the issue persists, please open a new issue in the ITKUltrasound repository.
